gems/audited/GHSA-hjp3-5g2q-7jww.yml

```diff
@@ -0,0 +1,37 @@
+---
+gem: audited
+ghsa: hjp3-5g2q-7jww
+url: https://github.com/collectiveidea/audited/security/advisories/GHSA-hjp3-5g2q-7jww
+title: Race Condition leading to logging errors
+date: 2023-05-01
+description: |
+  'In certain setups with threaded web servers, Audited's use of
```

Comment:
Should be safe to remove surrounding single-quotes from `|` block-text.

Reply:
fixed